Makefile

```diff
@@ -42,8 +46,20 @@ a11y: test-build pally-conf
 	cp -rf $(shell cat _test-server/template-copy-list.txt) bower_components/n-ui
 	PA11Y=true node _test-server/app
 
-test: verify pally-conf test-server test-unit test-build run nightwatch a11y
+# TODO: Figure this out: Because `run` never ends, `nightwatch` and `a11y` will never execute.
+# Unless ... are they execured in different processes? how does that work? Where's the output?
+test: developer-note verify pally-conf test-server test-unit test-build run nightwatch a11y
 
```

Comment:
Aha! I figured it out with @adgad — in CircleCI, `make run` exits when it is done, freeing the process so it can execute ` nightwatch a11y`.